react/src/components/atoms/forms/InputCheckBox/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Input from '../Input';
+import Icon from '../../icons/Icon';
+import { InputContext } from '../Input/context';
+import './style.css';
+
+const CheckBox = (props) => (
+  <React.Fragment>
+    <InputContext.Consumer>
+      {
+        (context) => {
+          const { id, value } = context;
+          const { icon, label, disabled } = props;
+          const handleClick = () => {
+            context.updateState({ value: !value }, () => {
+              if (typeof props.onChange === 'function') {
+                props.onChange(!value, id);
```

Comment:
While this is technically correct, I'd prefer to use getValue() to ensure that it's updated value:
```suggestion
                props.onChange(context.getValue(), id);
```